app/services/mlh_table.rb

```diff
@@ -28,11 +28,9 @@ def faraday_connection
         )
       end
     end
-    if response.success?
-      response.body
-    else
-      AirtablePlaceholderService.call('Meetups')
-    end
+    response.body if response.success?
+  rescue StandardError
```

Comment:
If `response.success?` is falsey, does this throw (so that the rescue can return the placeholder)? Or will this just return `nil` and rely on the downstream rescue in the helper?

Reply:
I tested this without the rescue in the helper, and it did throw.

Reply:
Right, but that means that this is returning nil and we're relying on the downstream rescue to save us -- the rescue here isn't doing anything?

I think we want to remove the rescue here (in MlhTable) and return the placeholder service (with the data key) if `response.success?` is falsey?

Reply:
If the rescue is removed from the helper, which is the rescue in play at that point? 

I had concluded that something else was required here, since without this rescue, and without the rescue in the helper, the app wasn't rendering any content (but was returning a `200`).

Reply:
If the helper rescue is removed, the remaining helper is the one here, in the service. I'm not sure this helper in the service does anything? I don't think faraday will throw, `response.success?` will just be falsey and we'll return nil. Instead of returning nil, we should return the placeholder.

Reply:
I'm wondering if this was a docker volume/eventual consistency thing at work – since now (after having rebuilt my local setup), the call to the placeholder service works without either the rescue in the helper or this rescue in the `faraday_connection` method. Before, this same combination wasn't returning any content at all (the same thing we were seeing before we added that StandardError rescue in the helper method). 
Anyhow, this finally seems to behave as expected, returning the placeholder without the rescue.